geode-core/src/integrationTest/java/org/apache/geode/management/internal/cli/GfshParserAutoCompletionTest.java

```diff
@@ -27,263 +27,266 @@
   }
 
   @Test
-  public void testJComplete3() throws Exception {
+  public void testJComplete3() {
     buffer = "start server --name=name1 --locators=localhost --J=-Dfoo=bar";
     candidate = parser.complete(buffer);
     assertThat(candidate.getCursor()).isEqualTo(buffer.length());
-    assertThat(candidate.size()).isEqualTo(51);
+    assertThat(candidate.getCandidates()).hasSize(51);
     assertThat(candidate.getFirstCandidate()).isEqualTo(buffer + " --assign-buckets");
   }
 
   @Test
-  public void testJComplete4() throws Exception {
+  public void testJComplete4() {
     buffer = "start server --name=name1 --locators=localhost  --J=-Dfoo=bar --";
     candidate = parser.complete(buffer);
     assertThat(candidate.getCursor()).isEqualTo(buffer.length() - 2);
-    assertThat(candidate.size()).isEqualTo(51);
+    assertThat(candidate.getCandidates()).hasSize(51);
     assertThat(candidate.getFirstCandidate()).isEqualTo(buffer + "assign-buckets");
   }
 
   @Test
-  public void testCompletRegionType() throws Exception {
+  public void testCompleteRegionType() {
     buffer = "create region --name=test --type";
     candidate = parser.complete(buffer);
-    assertThat(candidate.size()).isEqualTo(23);
+    assertThat(candidate.getCandidates()).hasSize(23);
     assertThat(candidate.getFirstCandidate()).isEqualTo(buffer + "=LOCAL");
   }
 
   @Test
-  public void testCompletPartialRegionType() throws Exception {
+  public void testCompletePartialRegionType() {
     buffer = "create region --name=test --type=LO";
     candidate = parser.complete(buffer);
-    assertThat(candidate.size()).isEqualTo(5);
+    assertThat(candidate.getCandidates()).hasSize(5);
     assertThat(candidate.getFirstCandidate()).isEqualTo(buffer + "CAL");
   }
 
   @Test
-  public void testCompletWithRegionTypeWithNoSpace() throws Exception {
+  public void testCompleteWithRegionTypeWithNoSpace() {
     buffer = "create region --name=test --type=REPLICATE";
     candidate = parser.complete(buffer);
-    assertThat(candidate.size()).isEqualTo(5);
+    assertThat(candidate.getCandidates()).hasSize(5);
     assertThat(candidate.getFirstCandidate()).isEqualTo(buffer + "_HEAP_LRU");
   }
 
   @Test
-  public void testCompletWithRegionTypeWithSpace() throws Exception {
+  public void testCompleteWithRegionTypeWithSpace() {
     buffer = "create region --name=test --type=REPLICATE ";
     candidate = parser.complete(buffer);
-    assertThat(candidate.size()).isEqualTo(45);
+    assertThat(candidate.getCandidates()).hasSize(45);
     assertThat(candidate.getFirstCandidate()).isEqualTo(buffer + "--async-event-queue-id");
   }
 
   @Test
-  public void testCompletLogLevel() throws Exception {
+  public void testCompleteLogLevel() {
     buffer = "change loglevel --loglevel";
     candidate = parser.complete(buffer);
-    assertThat(candidate.size()).isEqualTo(8);
+    assertThat(candidate.getCandidates()).hasSize(10);
```

Comment:
Was this an intentional change (8 changed to 10)? Now looks like it's a hard failure.

Reply:
Oops. Thanks for catching that. I cherry-picked and missed it.

Reply:
Fixed and pushed.